docs/docs/reference/language-guide/feel-context-expressions.md

```diff
@@ -5,15 +5,21 @@ title: Context Expressions
 
 ### Literal
 
-Creates a new context with the given entries. Each entry has a key and a value. The value can be any
-type.
+Creates a new context with the given entries. Each entry has a key and a value. The key is either a
+name or a string. The value can be any type.
```

Comment:
❓ Is it clear to readers what a `name` is? What characters can they use for names, and what about reserved identifiers?

Reply:
Good point :+1: 
I created a follow-up issue: #334